src/main/java/mate/academy/bookshop/model/Status.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.bookshop.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
+@Getter
+@Setter
+@ToString
+@EqualsAndHashCode
+@Table(name = "statuses")
+public class Status {

```

Comment:
We can have lots of different statuses across our application in the future
```suggestion
@Table(name = "order_statuses")
public class OrderStatus {
```